core/src/main/scala/spinal/core/HardMap.scala

```diff
@@ -0,0 +1,63 @@
+package spinal.core
+
+import spinal.idslplugin.Location
+
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+
+class HardMap extends MultiData {
+  val storage = mutable.LinkedHashMap[NamedType[Data], Data]()
```

Comment:
should be in another PR

Reply:
Ah yes, it's an accident that I pushed some other stuff... will do a repush